### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by sahlminkok**
### In this Pull Request we:
1. Add HTML code for the desktop version
2. Style the desktop version with CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,33 +78,40 @@ <h2 class="Description1">I'am a Software Developer</h2>
...
+            </p>
+            <ul class="list1">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a href="#" class="button">See project</a>
+          </div>
         </div>
```
Comment (osamaashraf6) [inline index.html:115]: - [ ] You did a great job 😃 on the project, but I see that your cards portfolio it is not been ordered on the page, so please try to organize the cards in the section to make it nice like the Figma 💯😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  image  |   Recording 2023-04-07 at 15 05 44  |

File: index.html [hunk 2]
```diff
@@ -66,33 +78,40 @@ <h2 class="Description1">I'am a Software Developer</h2>
...
+          /></a>
+        </li>
       </ul>
     </section>
     <section class="grid-container">
       <div class="main-div">
         <div class="top-div">
           <h2 id="MyRecentWorks">My Recent Works</h2>
           <img src="images/Vector.png" alt="horizontal-line" id="hr-line" />
+        </div>
```
Comment (osamaashraf6) [inline index.html:93]: - [ ] You did a great job 😃 on the project, but I see that the align of text here on the page not aligning with the cards, so please try to fix that to make it like the Figma Design 💯😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| image   |  image |

File: index.html [hunk 3]
```diff
@@ -177,48 +200,57 @@ <h3>Profesional Art <br />Printing Data</h3>
...
-      <h1>About me</h1>
-      <p>
-        Hello I’m a software developer! I can help you build a product , feature
-        or website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hesitate to contact me.
-      </p>
-      <a href="#" class="button">Get my resume</a>
+      <div class="illustration-about-me">
+        <h1>About me</h1>
+        <p>
```
Comment (osamaashraf6) [inline index.html]: - [ ] I see 😃 that you used the h1 tag twice in the page ,so please try to use the h just once in the page this is for the semantics and SEO ,so you can use hierarchy headings like h2, h3 ... to be friendly to the semantics 😊

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @sahlminkok 😊, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

SoCloseAlmostGIF

## Highlights🎉🎉
- Not linter errors present
- Good descriptive pull request
- Nice design 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (sahlminkok) [reply index.html]: @osamaashraf6 I have been trying so hard to make it aligned but unfortunately, it doesn't work

**Review by ShadyShawkat — APPROVED** [review]
## Hi @sahlminkok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**